location_extractor/containers.py

```diff
@@ -2,25 +2,27 @@
 from typing import List, Optional
 import abc
 
+from location_extractor.clients import LocationDTO
+
 
 class Entity(abc.ABC):
     @classmethod
     @abc.abstractmethod
-    def from_dict(cls, **kwargs):
+    def from_dto(cls, dto: LocationDTO):
         """
         Convert dictionary to Entity
-        :param kwargs: values for Entity fields
+        :param dto: LocationDTO
         :return: Entity
```

Comment:
we do not need to add typings in docstring, because we are using annotations for that